client/router_create.go

```diff
@@ -736,6 +736,10 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 // RouterCreate instantiates a VAN (router and controller) deployment
 func (cli *VanClient) RouterCreate(ctx context.Context, options types.SiteConfig) error {
 	// todo return error
+	if options.Spec.Ingress == types.IngressRouteString && cli.RouteClient == nil {
```

Comment:
@nicob87 One good addition would be to attach methods to the `SiteConfig` or `SiteConfigSpec`, like `IsIngressRoute`, `IsIngressLoadBalancer` and `IsIngressNone` and avoid comparing values directly.

Reply:
absolutelly! will do!

Reply:
Done